decentraland/screens/MyAccountScreen.js

```diff
@@ -27,46 +33,49 @@ const StepsWithAction = [
   },
 ];
 
-export class MyAccountScreen extends React.Component {
-  render() {
-    const { accountInfo } = this.props;
-    const { creationActions, account } = accountInfo;
-    const isAccountCreated = account !== null;
+const stepWasDone = (step, accountInfo) => {
+  const { creationActions, account } = accountInfo;
+  const { creationStatus } = step;
 
-    const creationSteps = [];
+  const statusWithAction = [
+    FUNDING_WITH_ETH,
+    FUNDING_WITH_MANA,
+    APPROVING_MARKETPLACE,
+  ];
 
-    creationSteps.push({
-      name: "Account created",
-      creationStatus: null,
-      status: isAccountCreated ? DONE : MISSING,
-      percentage: 0.25,
-    });
+  if (creationStatus === GENERATING_ACCOUNT) {
+    const isAccountCreated = account !== null;
+    return isAccountCreated;
+  } else if (statusWithAction.includes(creationStatus)) {
+    const hasAnAction = creationActions[creationStatus];
+    return hasAnAction;
```

Comment:
It's a little weird that sometimes the function returns a boolean and sometimes not.

Reply:
Both are treated as boolean outside, just make it's explicit.

Reply:
Resolved this

Reply:
The fix was related to this function.

Reply:
Ah ok interesting.